content/docs/features/bill-of-materials.md

```diff
@@ -0,0 +1,27 @@
++++
+title="Bill of Materials"
+summary="A Software `Bill-of-Materials` (`BOM`) gives you a layer-by-layer view of what's inside your container in a variety of formats including `JSON`."
++++
+
+## Summary
+A Software `Bill-of-Materials` (`BOM`) gives you a layer-by-layer view of what's inside your container in a variety of formats including `JSON`. Buildpacks can also populate your app image with metadata from the build process, allowing you to audit the app image for information like, the process types that are available and the commands associated with them, buildpacks which were used to create the app image etc. Apart from the above standard metadata, buildpacks can also populate information about the dependencies they have provided in form of a `Bill-of-Materials` (`BOM`).
```

Comment:
```suggestion
A Software **Bill-of-Materials** (`BOM`) provides information necessary to know what's inside your container and how it was constructed. 

Cloud Native Buildpacks provide two forms of Bill-of-Materials.

1. Buildpacks can populate `BOM` information about the dependencies they have provided.
2. A list of what buildpacks were used to build the application.
```

1. When we talk about the BoMs itself, I'm not sure that the format in which it is stored is too relevant.
2. The latter part of the paragraph seems to simply describe how buildpacks work and not necessarily related to BoMs

In my suggested changes I try to break down the information listed in `io.buildpacks.build.metadata`.
* `bom` is the buildpack populated bom
* `buildpacks` is the list of buildpacks.

See:

```
{
  "bom": [
    {
      "name": "some-world",
      "metadata": null,
      "buildpack": {
        "id": "samples/hello-world",
        "version": "0.0.1"
      }
    }
  ],
  "buildpacks": [
    {
      "homepage": "https://github.com/buildpacks/samples/tree/main/buildpacks/hello-world",
      "id": "samples/hello-world",
      "version": "0.0.1"
    }
  ],
  "launcher": {
    "version": "0.11.3",
    "source": {
      "git": {
        "repository": "github.com/buildpacks/lifecycle",
        "commit": "aa4bbac"
      }
    }
  },
  "processes": []
}
```